examples/structured_2d_dgsem/elixir_euler_warm_bubble.jl

```diff
@@ -0,0 +1,124 @@
+using OrdinaryDiffEq
+using Trixi
+
+# Physical constants
+g::Float64 = 9.81            # gravity of earth
+c_p::Float64 = 1004.0        # heat capacity for constant pressure (dry air)
+c_v::Float64 = 717.0         # heat capacity for constant volume (dry air)
+gamma::Float64 = c_p / c_v   # heat capacity ratio (dry air)
```

Comment:
Aha, I see that these constants are declared in a kind of global fashion.

Reply:
True. @ranocha already pointed this out to me. I understand it is not the best idea, but I somehow wanted to make sure that the same constant is used in the source term and the initial condition.

Reply:
Alternatively you could define a ["callable struct" (or functor)](https://docs.julialang.org/en/v1/manual/methods/#Function-like-objects) that holds these parameters and provides them for both IC and ST, e.g., something like this (not tested):
```julia
struct WarmBubbleSetup
    g::Float64     # gravity of earth
    c_p::Float64   # heat capacity for constant pressure (dry air)
    c_v::Float64   # heat capacity for constant volume (dry air)
    gamma::Float64 # heat capacity ratio (dry air)

    function WarmBubbleSetup(; g = 9.81, c_p = 1004.0, c_v = 717.0, gamma = c_p / c_v)
        new(g, c_p, c_v, gamma)
    end
end

# Initial condition
function (setup::WarmBubbleSetup(x, t, equations::CompressibleEulerEquations2D)
    @unpack g, c_p, c_v, gamma = setup
    ...
end

# Source terms
function (setup::WarmBubbleSetup(u, x, t, equations::CompressibleEulerEquations2D)
    @unpack g, c_p, c_v, gamma = setup
    ...
end

warm_bubble_setup = WarmBubbleSetup()

...

semi = SemidiscretizationHyperbolic(mesh, equations, warm_bubble_setup, solver,
                                    source_terms = warm_bubble_setup,
                                    boundary_conditions = boundary_conditions)

```

Reply:
Really nice! And (as usual) works without further ado. Thank you!